### PR TITLE
add integration with airtable

### PR DESCRIPTION
**Pull request by souzabrs**
script that read sos-rs.com API, transform the data a little bit and write to airtable

### CODE REVIEW

File: src/utils/integration/airtable.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  'Número de abrigados atualmente',
+  'Endereço',
+  'pix',
+  'Verificado',
+  'Aceita animais',
+  'Criado em',
+  'Atualizado em',
+];
+
+const TABLE_URI = '';
```
Comment (granbestiapop) [inline src/utils/integration/airtable.js:19]:
```suggestion
const TABLE_URI = '';
```
Move this to env var for better scalability 😀
Same case with for token.

File: src/utils/integration/airtable.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    const dataArr = [cols, ...Object.values(apiData)];
+
+    const csvData = stringify(dataArr);
+
+    const results = await writeToAirtable(csvData);
+
+    return JSON.stringify(results);
+  } catch (error) {
+    return error;
```
Comment (granbestiapop) [inline src/utils/integration/airtable.js:52]: Suggestion: log the error only. 
return empty results for fallback instead the error

File: src/utils/integration/airtable.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  }
+}
+
+async function mainLoop() {
+  // eslint-disable-next-line
+  while (true) {
```
Comment (granbestiapop) [inline src/utils/integration/airtable.js:58]: Suggestion: use set interval instead while true loop.

**Comment by souzabrs** [comment]
this script won't run at the backend, I'm just contributing to save and versionate the script, which is already running at a VM in GCP

also, this is a MVP integration, we don't have time to write perfect code. it's working and accomplishing it's goal

**Pull request closed without merging**